data/json/monsters/rodentkin.json

```diff
@@ -1,4 +1,45 @@
 [
+  {
+    "id": "mon_black_rat2",
```

Comment:
This should get an i explaining what it is, as of now it's too hard to tell what it is at a glance.